### PR TITLE
Add endpoint suffix for mysql and postgre sql

### PR DESCRIPTION
**Pull request by handsomesun**
**Description**  
Azure Postgre SQL and MySQL has different endpoint suffix than sql server on public cloud so creating separate configs for them.

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

**Comment by handsomesun** [comment]
> Could you also add support in the following part according to arm dict info?
> https://github.com/Azure/azure-cli/blob/c5661fa8410f6973cde67801953a4997e8961d4c/src/azure-cli-core/azure/cli/core/cloud.py#L187-L193

I just added

**Comment by Juliehzl** [comment]
@handsomesun As what I see in your description, you only mentioned that there are different endpoints for public cloud, but you also change China, US and German Cloud. 
Is it expected?

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -2534,14 +2529,14 @@
 type: command
 short-summary: Place the CLI in a waiting state until a condition of a scale set is met.
 examples:
-  - name: Place the CLI in a waiting state until a condition of a scale set is met. (autogenerated)
-    text: |
-        az vmss wait --created --name MyScaleSet --resource-group MyResourceGroup
-    crafted: true
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:2540]: Why change VM help?

**Comment by jiasli** [comment]
If the PR is targeting the wrong branch, edit it instead of creating a new PR. 

image

Also, if the commit history is not correct, [rebase](https://git-scm.com/docs/git-rebase) or [reset](https://git-scm.com/docs/git-reset) and [force push ](https://git-scm.com/docs/git-push#Documentation/git-push.txt---force) instead of creating a new PR.

**Comment by jiasli** [comment]
Continue working on #13115.

File: src/azure-cli/azure/cli/command_modules/vm/_help.py (hunk 1 above)
Comment (jiasli) [reply src/azure-cli/azure/cli/command_modules/vm/_help.py:2540]: This is due to wrong base branch.

**Pull request closed without merging**